### DISCUSSION TITLE
What is the RSS feed of a specific language of a project

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [WeblateOrg/weblate#7507] [Q&A] [answered]
The RSS icon on corresponding changes page just links to `https://hosted.weblate.org/exports/rss/weblate/`.

**Comment by nijel** [comment, marked as answer]
There are no language specific feeds, see https://hosted.weblate.org/data/weblate/

**Reply by Geeyun-JY3** [reply]
Any plan for this? Placing such an icon on the changes page may mislead users.

And I hope that there would be RSS feeds for a specific user. 

Furthermore, I also hope that adding different filters (project, language, component, action, author, starting date, ending date) to the current changes page would get different corresponding RSS feeds.

(Or I should open an issue for this?)

**Reply by nijel** [reply]
Opening issue will make sure this feature is not forgotten. On the other side, I don't see much use of the current RSS feeds, so investing time into that seems unlikely in the near future.

**Reply by Geeyun-JY3** [reply]
RSS feed can be used for notification. But most users would only involve in one language. Users don’t need notification for the activity of a whole project or a specific component (of all languages in a project). I think this is why current RSS feeds are not that useful. 
Users would only receive translation notifications (through email) of the language they involve in. They are similar.

For me, RSS feeds for a specific language of a project and a specific user are the most useful. 
The former can be used for notification as mentioned above. If adding different filters (project, language, component, action, author, starting date, ending date) to the current changes page would get different corresponding RSS feeds, then the RSS feeds can replace the email notification for me.
The latter can be used for getting my personal activity.
